src/eredis_client.erl

```diff
@@ -474,20 +476,24 @@ maybe_reconnect(Reason, #state{queue = Queue} = State) ->
 %% successfully issuing the auth and select calls. When we have a
 %% connection, give the socket to the redis client.
 reconnect_loop(Client, #state{reconnect_sleep=ReconnectSleep, transport=Transport}=State) ->
-    timer:sleep(ReconnectSleep),
-    case catch(connect(State)) of
-        {ok, #state{socket = Socket}} ->
-            Client ! {connection_ready, Socket},
-            Transport:controlling_process(Socket, Client),
-            Msgs = get_all_messages([]),
-            [Client ! M || M <- Msgs];
-        {error, _Reason} ->
-            reconnect_loop(Client, State);
-        %% Something bad happened when connecting, like Redis might be
-        %% loading the dataset and we got something other than 'OK' in
-        %% auth or select
-        _ ->
-            reconnect_loop(Client, State)
+    receive
+        {'EXIT', _, Reason} -> exit(Reason)
```

Comment:
Here `_` is `Client`, right? We could be explicit here to make it easier to understand the code. Or at least a comment about when/why this can happen.

Reply:
Good, done